src/Campaigns/Blocks/CampaignDonorsBlock/edit.tsx

```diff
@@ -0,0 +1,112 @@
+                <InspectorControls>
+                    <PanelBody title={__('Display Elements', 'give')} initialOpen={true}>
+                        <ToggleControl
+                            label={__('Show anonymous', 'give')}
+                            checked={showAnonymous}
+                            onChange={(value) => setAttributes({showAnonymous: value})}
+                        />
+                        <ToggleControl
+                            label={__('Show company name', 'give')}
+                            checked={showCompanyName}
+                            onChange={(value) => setAttributes({showCompanyName: value})}
+                        />
+                        <ToggleControl
+                            label={__('Show avatar', 'give')}
+                            checked={showAvatar}
+                            onChange={(value) => setAttributes({showAvatar: value})}
+                        />
+                        <ToggleControl
+                            label={__('Show button', 'give')}
+                            checked={showButton}
+                            onChange={(value) => setAttributes({showButton: value})}
+                        />
+                        <TextControl
+                            label={__('Donate Button', 'give')}
+                            value={donateButtonText}
+                            onChange={(value) => setAttributes({donateButtonText: value})}
+                            help={__('This shows on the header', 'give')}
+                        />
+                    </PanelBody>
+
+                    <PanelBody title={__('Settings', 'give')} initialOpen={true}>
+                        <SelectControl
+                            label={__('Sort by', 'give')}
+                            value={sortBy}
+                            options={[
+                                {label: __('Top Donors', 'give'), value: 'top-donors'},
+                                {label: __('Recent Donors', 'give'), value: 'recent-donors'},
+                            ]}
+                            onChange={(value) => setAttributes({sortBy: value})}
+                            help={__('The order donors are displayed in.', 'give')}
+                        />
+                        {/* TODO: Revert the label and help text back to what are in the designs once the backend for pagination is ready */}
+                        <NumberControl
+                            label={__('Limit', 'give')}
+                            value={donorsPerPage}
+                            min={1}
+                            max={100}
+                            onChange={(value) => setAttributes({donorsPerPage: parseInt(value)})}
+                            help={__('The maximum number of donors to display.', 'give')}
+                        />
+                        {/* TODO: Revert the field back once the backend for pagination is ready
```

Comment:
Same here

Reply:
These comments are part of the changes I made while temporarily disabling pagination (load more button) on this block. It's a comment for the future.